snapcraft/__init__.py

```diff
@@ -225,6 +225,18 @@
     `prime/` directory reflects the file structure of the snap with no
     extraneous content).
 
+  - prepare: shell script
+
+    If present, the shell script defined here is run before the `build` step
+    of the plugin starts. The working directory is the base build
+    directory for the given part. The defined script is run with `/bin/sh`.
```

Comment:
I just realized: `/bin/sh` will be different for different people, which means the same `snapcraft.yaml` will succeed for some people, and fail for others. Should we instead be using a specific shell?